File: sglimu-test1-f7f80df16df9dd0d0a27af85970c4fb2f22e7ee0/scratch/db_test.py
import urllib.request
import json

url = 'https://xxvfgnoffomrhtxitqkj.supabase.co/rest/v1/orders?select=*&limit=1'
headers = {
    'apikey': 'sb_publishable_Q4t2p9WcUBdtUxd7HYV56A_MvxnZRk9',
    'Authorization': 'Bearer sb_publishable_Q4t2p9WcUBdtUxd7HYV56A_MvxnZRk9'
}

req = urllib.request.Request(url, headers=headers)
try:
    with urllib.request.urlopen(req) as response:
        html = response.read().decode('utf-8')
        data = json.loads(html)
        print("Success fetching orders:", data)
except Exception as e:
    print("Error fetching orders:", e)

File: sglimu-test1-f7f80df16df9dd0d0a27af85970c4fb2f22e7ee0/scratch/db_test.js
const { createClient } = require('@supabase/supabase-js');
const SUPABASE_URL = 'https://xxvfgnoffomrhtxitqkj.supabase.co';
const SUPABASE_ANON_KEY = 'sb_publishable_Q4t2p9WcUBdtUxd7HYV56A_MvxnZRk9'; // test_supabase.js에 적혀있던 키 사용
const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY);

async function checkOrdersSchema() {
    const { data, error } = await supabase.from('orders').select('*').limit(1);
    if (error) {
        console.error('Error fetching orders:', error);
    } else {
        console.log('Orders row sample:', data);
    }
}
checkOrdersSchema();
